python/_fio_plugin/.github/build.yaml

```diff
@@ -0,0 +1,17 @@
+name: Build
```

Comment:
Will this work in the subdirectory?

Reply:
I have no idea. I only added it b/c @janosdebugs suggested it in his earlier review